scripts/translations/invalidateOutdated.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+
+// This script removes translations from all the non-English translation JSON files for
+// entries which have been modified in English since a specific Git revision.
+// The new and deleted English copies are ignored as they will be automatically handled
+// by Gitlocalize, our translation platform.
+
+import { execSync } from 'child_process'
+import * as fs from 'fs'
+import readline from 'readline'
+
+import * as utils from './utils'
+
+// Set the current dir to the repo root (makes it easier for git commands)
+const repoPath: string = execSync('git rev-parse --show-toplevel').toString().trim()
+process.chdir(repoPath)
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+let commitId: string
+if (process.argv.length > 2) {
+  let commitId = process.argv.splice(2, process.argv.length - 1).join(' ')
```

Comment:
No need to redeclare.

```suggestion
  commitId = process.argv.splice(2, process.argv.length - 1).join(' ')
```

Reply:
Indeed, a bit of a careless copy/paste on my part from here: https://github.com/shapeshift/web/blob/00c817051c127d628ab5b51a661a02790351a1e7/scripts/translations/countEnglishWordsSince.ts#L21

Thanks for the review, I'll make the change in a new PR 😅